content/livesync/models/models.textile

```diff
@@ -58,7 +58,7 @@ The following is an example response from the sync function:
 The sync function needs to be registered when a model is "instantiated":#create, for example:
 
 ```[javascript]
-const modelsClient = modelsClient.models.get({
+const myPostModel = modelsClient.models.get({
```

Comment:
I think we need to make it `model`, i.e.
```
const model = modelsClient.models.get ...
```
Because later in the docs we're referring to it as `model` in ```await model.sync(123, 1);```

Reply:
Yep either that or later we do `await myPostModel.sync(123, 1)`

Reply:
I'll amend with `const model = modelsClient.models.get ...`

@snikidev and @mschristensen 

👍

Reply:
@mschristensen @snikidev, please see: b6820c9945d05589490d4499e68f9a07a059d000